geopolars/Cargo.toml

```diff
@@ -6,8 +6,9 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-arrow2 = {version = "0.11.2", features = ["io_ipc"]}
-geo = "0.20.1"
-geozero = {version = "0.9.4", features = ["with-wkb"]}
-polars = {version = "0.21.1", features = ["ipc", "dtype-u8", "dtype-i8"]}
+arrow2 = { version = "0.11.2", features = ["io_ipc"] }
+geozero = { version = "0.9.4", features = ["with-wkb"] }
 rstar = "0.9.3"
+geo = { git = "https://github.com/georust/geo.git", rev = '578e213875915e1f895892487b5a36ca0d91fac3' }
+polars = { version = "0.21.1", features = ["ipc", "dtype-u8", "dtype-i8"] }
+geo-types = { version = "0.7.6", features = ["approx", "use-rstar_0_9"] }
```

Comment:
Is a direct dependency on `geo-types` the standard way to include `rstar`?

Reply:
It's not for the r-star dep it's for the affine transform support that just landed. Once that gets released we can move back to the a versioned number. Can prob take the r-star part out.

Reply:
Actually I was able to remove this just fine without issue